### PR TITLE
Fluid mobile font and viewport breakpoint standardization

### PR DESCRIPTION
**Pull request by fan-matt**
Basically fixes the issue where headers wrap and the text "making coding ..." is too large and pushes against the size of its container.

All it does is makes the text depend on viewport width when viewing in mobile and reverts to original sizing when viewing on a desktop.

Happy Thanksgiving yall!

Edit: Also changes all viewport breakpoints to a sass variable set at 850px

### CODE REVIEW

**Comment by fan-matt** [comment]
> Another solution could just be to change the column viewport breakpoint, so the homepage is single-column at a larger viewport - in that case, I don't think there will be many issues w/ text wrapping and the like.

I set the breakpoint to 850px- that way, the words don't dip into the padding on the sides. Sorry for the late reply, let me know what you think.

File: _sass/_basic.scss [hunk 1]
```diff
@@ -38,9 +38,22 @@
...
+    font-size: 5.5vw;
+
+    @media (min-width: $desktop-width) {
+        font-size: 1.5em;
+    }
+}
+
 .page-title{
-    font-size: 3em;
+    // font-size: 3em;
```
Comment (malsf21) [inline _sass/_basic.scss]: Can you remove this comment? Other than that, looks good to merge!

Comment (fan-matt) [reply _sass/_basic.scss]: sorry! I'll do that. Do you still want the comments in variables.scss?

**Comment by malsf21** [comment]
Actually, one other favour too - can you merge master into your branch, and then apply the SASS variable to the breakpoint in `team-content` I think so Arjun's name wraps properly?

File: _sass/_basic.scss (hunk 1 above)
Comment (malsf21) [reply _sass/_basic.scss]: Oh no those also aren't needed, I guess I forgot to delete them when I ported over our ACM stuff!

**Comment by fan-matt** [comment]
> Actually, one other favour too - can you merge master into your branch, and then apply the SASS variable to the breakpoint in `team-content` I think so Arjun's name wraps properly?

Should I go ahead and replace with the variable everywhere?

**Comment by malsf21** [comment]
> Should I go ahead and replace with the variable everywhere?

Yeah, for any portion of the website that relies on the value for the collapse breakpoint. 

Also you might want to rename the PR/description to include this change :P

**Comment by fan-matt** [comment]
will do, I'm off to eat but I'll be back :)

**Comment by fan-matt** [comment]
Hey Matt, looks like it's at a point where it's ready to merge. I also fixed a bug where the team member picture is slightly smaller if the name is two lines long. Let me know what you think

**Pull request merged**